code/game/machinery/portable_turret.dm

```diff
@@ -902,7 +902,7 @@ Status: []<BR>"},
 
 /obj/machinery/porta_turret_cover/attack_hand(mob/user)
 	. = ..()
-	if(.)
+	if(. || !Parent_Turret)

```

Comment:
I suspect this is occurring when the main turret has just been GC'd but someone manages to get a click in just before the cover goes with it. There are probably checks for that, but there's always edge cases.